rpmbuild/copr-rpmbuild.spec

```diff
@@ -139,8 +139,14 @@ Requires: yum-utils
 %latest_requires ca-certificates
 %latest_requires distribution-gpg-keys
 %if 0%{?fedora}
+
+%if 0%{?fedora} >= 39
+%latest_requires dnf5
```

Comment:
I'm not sure if we are fully compatible with DNF5.  In Mock and prunerepo we switch to /bin/yum-3 and `python3-dnf`.  Would it make sense to do the same thing here?

Reply:
I am not sure but, I think we don't care.
The `copr-rpmbuild` doesn't do anything with DNF, this is just `copr-builder` dependency to ensure we have the latest tooling on builders.

So if `mock` can run on a system that has DNF5 (can it?), we should be fine.

Reply:
Ah, you are right.  Then we probably want %latest_requires for both `dnf5` and `python3-dnf`.
The newer chroots will probably prefer DNF5, older DNF4 (`dnf` is misleading, because it may be any of those).

Reply:
The same way we'll probably need `dnf5-plugins` (for builddep, repoquery, etc...)

Reply:
Note to me: Don't forget on `dnf5-plugins`

Reply:
Updated, PTAL